iroh-net/src/netcheck/reportgen.rs

```diff
@@ -366,9 +366,9 @@ impl Actor {
                 Probe::Https { .. } | Probe::Icmp { .. } => (),
             }
         }
-        self.report.ipv4_can_send = probe_report.ipv4_can_send;
-        self.report.ipv6_can_send = probe_report.ipv6_can_send;
-        self.report.icmpv4 = probe_report.icmpv4;
+        self.report.ipv4_can_send |= probe_report.ipv4_can_send;
```

Comment:
I think this is wrong, if our report says we cant send anymore we shouldn’t keep this true

Reply:
Here the `probe_report` is of a single probe: the `Ipv6Stun` probe will always report `ipv4_can_send` to be false, just like the `Ipv4Stun` probe will always report `ipv6_can_send` to be false.  As we assemble the entire netcheck report we need to combine these results from individual probes together, without trampling over each other's results.  So I believe this is correct.

Reply:
ahh okay, you are right